src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -110,34 +120,40 @@ const ActivityThread = ({
     }
     return (
         <div className="bcs-ActivityThread" data-testid="activity-thread">
-            <div className="bcs-ActivityThread-content">
-                {children}
+            <div className="bcs-ActivityThread-selectWrapper">
```

Comment:
Is this needed or can we style `bcs-ActivityThread` directly?

Reply:
Unfortunately it's needed to meet the design.